exec/mem.go

```diff
@@ -49,6 +49,11 @@ func NewMemCommandModelSpec() spec.ExpModelCommandSpec {
 					Desc:     "percent of burn Memory (0-100)",
 					Required: false,
 				},
+				&spec.ExpFlag{
+					Name:     "reserve",
+					Desc:     "reserve to burn Memory, unit is MB",
```

Comment:
I see that the `--mem-percent` and `--reserve ` flag only one take effect.  Is it necessary to add it to the flag description and indicate the priority?